### DISCUSSION TITLE
ListView - disable "type to focus"?

### DISCUSSION THREAD

**Discussion by aloneguid** [tui-cs/Terminal.Gui#3016] [Q&A]
I'm not sure how to call this feature - when I type in list view, it will focus on the item by letter i type. Whereas it's useful in most cases, I'd like to completely disable this for one ListView and key events to bubble up to parent view. Is this possible?

**Comment by tznind** [comment]
Please try:
```
myListView.KeystrokeNavigator.Collection = Enumerable.Empty<string> ().ToList();
```
